src/Parser/ClassParser.php

```diff
@@ -190,11 +190,21 @@ private function getParentClassStatements(): ?array
             return [];
         }
 
-        if (null !== $usedClasses[$this->getParentClassName()]) {
+        if (null !== @$usedClasses[$this->getParentClassName()]) {
```

Comment:
```suggestion
        if (true === isset($usedClasses[$this->getParentClassName()])) {
```
I think this would solve this as well :v: